### PR TITLE
[WAF] Revert Security Level updates

### PR DESCRIPTION
**Pull request by pedrosousa**
Related to PCX-10665.

This reverts commit 387b4eafc82c66f01e629997fd897c7939d013a8.

### CODE REVIEW

File: content/terraform/tutorial/add-page-rules.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ cloudflare_page_rule.increase-security-on-expensive-page: Creation complete afte
 Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
 ```
 
-With the Page Rules in place, try that call again, along with a test for the I'm Under Attack mode:
+With the Page Rules in place, try that call again, along with a test for the "I Am Under Attack" mode:
```
Comment (angelampcosta) [inline content/terraform/tutorial/add-page-rules.md]: I think you used `I'm` before.

File: content/terraform/tutorial/add-page-rules.md [hunk 2]
```diff
@@ -213,4 +213,4 @@ $ curl -vso /dev/null https://www.example.com/expensive-db-call 2>&1 | grep "< H
 < HTTP/1.1 503 Service Temporarily Unavailable
 ```
 
-The call works as expected. In the first case, the Cloudflare global network responds with a `301` redirecting the browser to the new location. In the second case, the Cloudflare global network initially responds with a `503`, which is consistent with the I'm Under Attack mode.
+The call works as expected. In the first case, the Cloudflare global network responds with a `301` redirecting the browser to the new location. In the second case, the Cloudflare global network initially responds with a `503`, which is consistent with the "I Am Under Attack" mode.
```
Comment (angelampcosta) [inline content/terraform/tutorial/add-page-rules.md]: Same here.

File: content/learning-paths/application-security/default-traffic-security/ddos.md [hunk 3]
```diff
@@ -5,8 +5,8 @@ weight: 1
...
 ---
 
-Cloudflare automatically detects and mitigates DDoS attacks using its [Autonomous Edge](/ddos-protection/about/components/#autonomous-edge), which is always-on. `Advanced` protections are reserved for Magic Transit customers.
+Cloudflare automatically detects and mitigates DDoS attacks using its [Autonomous Edge](/ddos-protection/about/components/#autonomous-edge), which is always-on. `Advanced` protections are reserved for Magic Transit customers. 
 
 {{<render file=_ddos-attack-coverage.md productFolder="ddos-protection">}}
 
-Refer to the learning path [Prevent DDoS attacks](/learning-paths/prevent-ddos-attacks/) to dive deeper into this subject.
\ No newline at end of file
+Refer to the learning path [`Prevent DDoS Attaacks`](/learning-paths/prevent-ddos-attacks/)  to dive deeper into this subject.
```
Comment (angelampcosta) [inline content/learning-paths/application-security/default-traffic-security/ddos.md]: I don't think we use backticks for learning path's names.

**Review by angelampcosta — APPROVED** [review]

File: content/terraform/tutorial/add-page-rules.md (hunk 1 above)
Comment (pedrosousa) [reply content/terraform/tutorial/add-page-rules.md]:
```suggestion
With the Page Rules in place, try that call again, along with a test for the I'm Under Attack mode:
```

File: content/terraform/tutorial/add-page-rules.md (hunk 2 above)
Comment (pedrosousa) [reply content/terraform/tutorial/add-page-rules.md]:
```suggestion
The call works as expected. In the first case, the Cloudflare global network responds with a `301` redirecting the browser to the new location. In the second case, the Cloudflare global network initially responds with a `503`, which is consistent with the I'm Under Attack mode.
```

File: content/learning-paths/application-security/default-traffic-security/ddos.md (hunk 3 above)
Comment (pedrosousa) [reply content/learning-paths/application-security/default-traffic-security/ddos.md]:
```suggestion
Refer to the learning path [Prevent DDoS attacks](/learning-paths/prevent-ddos-attacks/)  to dive deeper into this subject.
```

**Pull request merged**